installer/resources/lambda_submit/function.py

```diff
@@ -804,7 +804,7 @@ class LongRunningLambdaFunction(LambdaFunctionResource):
     handler = BATCH_LONG_RUNNING_JOB_FILE_NAME + ".lambda_handler"
     runtime = "python3.8"
     s3_bucket = BucketStorage.get_output_attr('bucket')
-    s3_key = UploadLambdaSubmitJobZipFile.get_output_attr('id') 
+    s3_key = UploadLambdaLongRunningJobZipFile.get_output_attr('id') 
```

Comment:
How is this related to the data collectors?

Reply:
DataColletor lambda is picking the wrong code from s3.